### PR TITLE
More examples...

### PR DESCRIPTION
**Pull request by kshyatt**
Hopefully this has less branch horror than last time I tried this!

### CODE REVIEW

**Comment by matbesancon** [comment]
check passing, already a good sign

File: src/core.jl [hunk 1]
```diff
@@ -109,10 +177,10 @@ end
 """
     degree_histogram(g, degfn=degree)
 
-Return a Dict with values representing the number of vertices that have degree
+Return a `Dict` with values representing the number of vertices that have degree
```
Comment (matbesancon) [inline src/core.jl]: should we use 3rd person for docs? Like "Returns" ?

Comment (kshyatt) [reply src/core.jl]: `Return` is base Julia style

File: src/core.jl [hunk 2]
```diff
@@ -146,6 +238,31 @@ and [`inneighbors`](@ref).
...
 ### Implementation Notes
 Returns a reference, not a copy. Do not modify result.
+
+# Examples
+```jldoctest
+julia> using LightGraphs
+
+julia> g = DiGraph();
+
+julia> add_vertices!(g, 3);
```
[the remark this answers is not in the document]
Comment (kshyatt) [reply src/core.jl]: just can't get enough of adding vertices or comments 😆

**Pull request merged**